stable/luigi/values.yaml

```diff
@@ -83,3 +83,7 @@ mysql:
   mysqlAllowEmptyPassword: true
   persistence:
     enabled: false
+
+podDisruptionBudget:
+  enabled: true
```

Comment:
I would say this should be false by default

Reply:
thanks, I've pushed a corresponding change with squashed commit.